<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>query</class>
 <widget class="QWidget" name="query">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="queryWord">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="precise">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>精确查找</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">precisionGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="vague">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>140</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>模糊查找</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">precisionGroup</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>查询关键字</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>921</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>140</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>按商品名称</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按进货/出货</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按联系人</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按产地</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按交易地点</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按交易单位</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>查询类别</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="precisionGroup"/>
 </buttongroups>
</ui>
